Gemfile

```diff
@@ -12,3 +10,6 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 # your gemspec. These might include edge Rails or gems from your path or
 # Git. Remember to move these dependencies to your gemspec before releasing
 # your gem to rubygems.org.
+
+# Load Gemfile with dependencies from dummy app
+eval_gemfile(File.expand_path("spec/dummy/Gemfile", __dir__))
```

Comment:
This is another point of discussion that I'm not sure about, and I'm honestly not sure how it even worked before.  Previously we had the following changes that I did not bring over:

https://github.com/ManageIQ/manageiq-schema/pull/616/files#diff-34e1b7b0905d8d8ffb401395c2e3bb17ee91c912b307a7586583c3ad132684b6L2-R1 and https://github.com/ManageIQ/manageiq-schema/pull/616/files#diff-ad366e89306810a068ae008a627d8fe6c27f88bf77bee35cd828f784bc6de062L14

Notice the paths - Previously they were `../../..` and now they are `..`.  This implies that the dummy app reached back into the plugin and used its Gemfile, but I don't understand how that could have possibly worked.  In fact, when I first upgraded to Rails 6, none of the rake tasks were available to me, such as `db:drop`, and I don't understand how it was possible before.

---

I think this change here, that uses the dummy's Gemfile in the way we do in ManageIQ is a more accurate reflection of what we do with other plugins.  There might be a more "correct" way of testing plugins with dummy apps, but I haven't found it yet.

Reply:
Decided to do it the old way for now, which seems to work, so as not to change too many things at once.